src/m365/spfx/commands/project/project-github-workflow-add.ts

```diff
@@ -190,9 +181,7 @@ class SpfxProjectGithubWorkflowAddCommand extends BaseProjectCommand {
       this.getDeployAction(workflow).with!.SKIP_FEATURE_DEPLOYMENT = true;
     }
 
-    if (options.overwrite) {
-      this.getDeployAction(workflow).with!.OVERWRITE = true;
-    }
+    this.getDeployAction(workflow).with!.OVERWRITE = true;
```

Comment:
actually this is not how we should go about it. We should totally remove this line and changing this property from the command logic 
```suggestion
```

Instead we should set the default `true` in the JSON representation of the scaffolded workflow here

https://github.com/pnp/cli-microsoft365/blob/bf59841402cda9e3766ee4b7ae89fe12a4d57361/src/m365/spfx/commands/project/DeployWorkflow.ts#L52